modules/generator/instance.go

```diff
@@ -447,10 +447,29 @@ func (i *instance) queryRangeTraceQLToProto(set traceql.SeriesSet, req *tempopb.
 			})
 		}
 
+		exemplars := make([]tempopb.Exemplar, 0, len(s.Exemplars))
```

Comment:
It looks like already added this to `SeriesSet.ToProto`.  Maybe this entire func is obsolete and can be replaced with a call to that?

Reply:
It was. I've removed it